forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt

```diff
@@ -0,0 +1,11 @@
+Name:Breeches, the Blastmaker
+ManaCost:1 U R
+Types:Legendary Creature Goblin Pirate
+PT:3/3
+K:Menace
+T:Mode$ SpellCast | ValidCard$ Card.YouCtrl | TriggerZones$ Battlefield | Execute$ TrigFlip | ValidActivatingPlayer$ You | ActivatorThisTurnCast$ EQ2 | TriggerDescription$ Whenever you cast your second spell each turn, you may sacrifice an artifact. If you do, flip a coin. When you win the flip, copy that spell. You may choose new targets for the copy. When you lose the flip,CARDNAME deals damage equal to that spell's mana value to any target.
+SVar:TrigFlip:AB$ FlipACoin | Cost$ Sac<1/Artifact> | WinSubAbility$ DBCopy | LoseSubAbility$ DBDamage
+SVar:DBCopy:DB$ CopySpellAbility | Defined$ TriggeredSpellAbility | MayChooseTarget$ True | SpellDescription$ When you win the flip, copy that spell. You may choose new targets for the copy.
+SVar:DBDamage:DB$ DealDamage | ValidTgts$ Any | NumDmg$ X | SpellDescription$ When you lose the flip, CARDNAME deals damage equal to that spell's mana value to any target.
```

Comment:
I think this needs to be wrapped in an immediatetrigger ?

Reply:
yes

Reply:
Same with the one that copies the spell ?

Reply:
yes

Reply:
Hmm I can't seem to find the correct way to remember the triggeredobject on the copy

Reply:
Use Spawner Syntax instead

Reply:
I don't understand why you switched the triggeringObject to Card

Reply:
What should the correct syntax be in this case ?

Reply:
you want to copy a spell and not a card, so TriggeredSpellAbility like always

Reply:
Yeah I've tried this before but with no succes

Reply:
now use it for both

Reply:
I see it doesn't work fully yet, will make a fix

Reply:
Thank you !